mllib/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -1 +1,2 @@
 org.apache.spark.ml.source.libsvm.LibSVMFileFormat
+org.apache.spark.ml.source.image.ImageFileFormat
```

Comment:
tiny nit: newline at the end - this might be caught by maven's checkstyle (https://github.com/apache/spark/pull/21801)